sdks/python/apache_beam/io/gcp/bigquery_read_it_test.py

```diff
@@ -69,6 +70,20 @@ def wrapped(self):
   return inner
 
 
+def datetime_to_utc(element):
+  for k, v in element.items():
+    if isinstance(v, datetime.time):
```

Comment:
We can combine this condition with the one below:
`if isinstance(v, datetime.time) or isinstance(v, datetime.date):`

Reply:
or `isinstance(v, (datetime.time, datetime.date))`